packages/vm/test/api/istanbul/eip-152.deactivated.ts

```diff
@@ -97,7 +97,7 @@ describe('Istanbul: EIP-152', () => {
       const res = precompile09({
         data: hexToBytes('0x' + testCase.input),
         gasLimit: BigInt(20),
-        _common: common,
+        common: common,
```

Comment:
```suggestion
        common,
```

Super nit but we can now do this

Reply:
Will ignore this for now, we should really activate linter for test files though, this would then be caught and fixed.